images/build/Dockerfile

```diff
@@ -15,7 +15,7 @@ ENV LANG C.UTF-8
 
 # Install basic build tools
 RUN apt-get update \
-    && apt-get upgrade -y \
+#    && apt-get upgrade -y \
```

Comment:
Why these changes?

Reply:
just FYI...we added the upgrade to avoid getting flagged by S360 for missing security updates...